src/ansys/acp/core/_tree_objects/cut_off_selection_rule.py

```diff
@@ -63,50 +63,50 @@
 
 
 __all__ = [
-    "CutoffSelectionRule",
-    "CutoffSelectionRuleElementalData",
-    "CutoffSelectionRuleNodalData",
+    "CutOffSelectionRule",
+    "CutOffSelectionRuleElementalData",
+    "CutOffSelectionRuleNodalData",
 ]
 
 
 @dataclasses.dataclass
-class CutoffSelectionRuleElementalData(ElementalData):
-    """Represents elemental data for a Cutoff Selection Rule."""
+class CutOffSelectionRuleElementalData(ElementalData):
+    """Represents elemental data for a Cut Off Selection Rule."""
 
     normal: VectorData | None = None
 
 
 @dataclasses.dataclass
-class CutoffSelectionRuleNodalData(NodalData):
-    """Represents nodal data for a Cutoff Selection Rule."""
+class CutOffSelectionRuleNodalData(NodalData):
+    """Represents nodal data for a Cut Off Selection Rule."""
 
 
 @mark_grpc_properties
 @register
-class CutoffSelectionRule(CreatableTreeObject, IdTreeObject):
-    """Instantiate a Cutoff Selection Rule.
+class CutOffSelectionRule(CreatableTreeObject, IdTreeObject):
+    """Instantiate a Cut Off Selection Rule.
```

Comment:
I would use Cut-Off and Drop-Off in doc strings to be consistent with the ACP documentation.
```suggestion
    """Instantiate a Cut-Off Selection Rule.
```